### ISSUE TITLE
Conformance processing continues even if no conformance statement

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#517]
This is the same issue I had with the epub techniques (but is now fixed).

After the first set of if/else checks for the general conformance statement, processing continues even if the result was that no information was specified. The result is that, at a minimum, you'll get an empty header for the detailed conformance. If there is no conformance information, all the processing should be skipped.

**Comment by clapierre** [comment]
fixed. (we think :)
